example/example/settings.py

```diff
@@ -123,3 +125,6 @@
 
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 MEDIA_URL = '/media/'
+
+FIO_BANK_TOKEN = 'token'
+PAYMENT_ERROR_RATE = Decimal('0.0')
```

Comment:
consider integer field: 0 to 100 as that is also used in DB for discount (so we are more consistent, and is more user friendly to setup). Originally I thought we can have 3.5% error rate...